gateway/conf/nginx.conf.liquid

```diff
@@ -5,12 +5,16 @@ env RESOLVER;
 env BACKEND_ENDPOINT_OVERRIDE;
 env OPENSSL_VERIFY;
 
+{% if jaeger_enabled %}
+load_module /opt/app-root/src/nginx-modules/ngx_http_opentracing_module.so;
+load_module /opt/app-root/src/nginx-modules/ngx_http_jaeger_module.so;
```

Comment:
This will not work on macOS or any local machine. We have to make this relative somehow.